stt:
  default_param:
    language_id: "it-IT" #"en-US"
    sample_rate: 16000 #44100 for wav audio file, 16000 for wav file from mic
    audio_channel: 1
    subscriber_id: "default"
    credential_path: "/root/harmoni_catkin_ws/src/HARMONI/dockerfiles/config/private-keys.json" #path where private keys are mounted
  ita_param:
    language_id: "it-IT"
    sample_rate: 16000 #44100 for wav audio file
    audio_channel: 1
    subscriber_id: "default"
    credential_path: "/root/harmoni_catkin_ws/src/HARMONI/dockerfiles/config/private-keys.json" #path where private keys are mounted
